src/Og.php

```diff
@@ -169,6 +169,32 @@ public static function getEntityGroups($entity_type, $entity_id, $states = [OG_S
   }
 
   /**
+   * Return TRUE if entity belongs to a group.
+   *
+   * @param string $group_entity_type
+   *   The entity type of the group.
+   * @param string|int $group_id
+   *   The group ID.
+   * @param string $entity_type
+   *   The entity type.
+   * @param string|int $entity_id
+   *   The entity ID.
+   * @param array $states
+   *   (optional) Array with the state to return. If empty groups of all state
+   *   will return.
+   *
+   * @return bool
+   *   TRUE if the entity (e.g. the user) belongs to a group and is not pending
+   *   or blocked.
+   */
+  public static function isMember($group_entity_type, $group_id, $entity_type, $entity_id = NULL, $states = [OG_STATE_ACTIVE]) {
```

Comment:
Yeah I would Def prefer this but just kept it the same. I am happy to
change it here though.
On 30 Nov 2015 18:11, "Amitai Burstein" <notifications@github.com> wrote:

> In src/Og.php
> <https://github.com/amitaibu/og/pull/53#discussion_r46179638>:
>
> > +   *   The entity type of the group.
> > +   * @param string|int $group_id
> > +   *   The group ID.
> > +   * @param string $entity_type
> > +   *   The entity type.
> > +   * @param string|int $entity_id
> > +   *   The entity ID.
> > +   * @param array $states
> > +   *   (optional) Array with the state to return. If empty groups of all state
> > +   *   will return.
> > +   *
> > +   * @return bool
> > +   *   TRUE if the entity (e.g. the user) belongs to a group and is not pending
> > +   *   or blocked.
> > +   */
> > +  public static function isMember($group_entity_type, $group_id, $entity_type, $entity_id = NULL, $states = [OG_STATE_ACTIVE]) {
>
> I think maybe it's a good time to replace the arguments with the entity
> objects, so it will be isMember(EntityInterface $group_entity,
> EntityInterface $entity) instead. Thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amitaibu/og/pull/53/files#r46179638>.
>